### PR TITLE
Fix TTS FastPitch tutorial

### PR DESCRIPTION
**Pull request by hsiehjackson**
# What does this PR do ?

Add a one line overview of what this PR aims to accomplish.

**Collection**: [Note which collection this PR will affect]

# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# Before your PR is "Ready for review"
**Pre checks**:
- [x] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [x] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

File: tutorials/tts/FastPitch_Adapter_Finetuning.ipynb [hunk 1]
```diff
@@ -100,7 +100,7 @@
     "# .nemo files for your pre-trained FastPitch and HiFiGAN\n",
     "pretrained_fastpitch_checkpoint = \"<Multispeaker pretrained checkpoint path.>\"\n",
     "finetuned_hifigan_on_multispeaker_checkpoint = \"<Pretrained hifiGan checkpoint path.>\"\n",
-    "use_ipa = True #Set to False while using Arpabet."
+    "use_ipa = False #Set to False while using Arpabet."
```
Comment (XuesongYang) [inline tutorials/tts/FastPitch_Adapter_Finetuning.ipynb:103]: Does it matter if flipping the `use_ipa` for the bug?

File: tutorials/tts/FastPitch_Adapter_Finetuning.ipynb [hunk 2]
```diff
@@ -612,7 +612,7 @@
     "model.optim.lr=0.0001 \\\n",
     "model/train_ds=train_ds_finetune \\\n",
     "model/validation_ds=val_ds_finetune \\\n",
-    "+trainer.max_epochs=500 \\\n",
+    "+trainer.max_epochs=50 \\\n",
```
Comment (XuesongYang) [inline tutorials/tts/FastPitch_Adapter_Finetuning.ipynb:615]: nit: to confirm if these changes are intentional.

**Review by XuesongYang — CHANGES_REQUESTED** [review]
removing `trainer.strategy` is not enough, because your code will use the default `ddp` as in https://github.com/NVIDIA/NeMo/blob/main/examples/tts/conf/fastpitch_align_44100_adapter.yaml#L291. It would be better to specify it as `auto` I think.

File: tutorials/tts/FastPitch_Adapter_Finetuning.ipynb (hunk 1 above)
Comment (hsiehjackson) [reply tutorials/tts/FastPitch_Adapter_Finetuning.ipynb:103]: Yes, we use the pre-trained chpt without ipa to test the tutorial.

**Comment by XuesongYang** [comment]
it looks like the similar issue as here: https://github.com/NVIDIA/NeMo/pull/7509

File: tutorials/tts/FastPitch_Adapter_Finetuning.ipynb (hunk 2 above)
Comment (hsiehjackson) [reply tutorials/tts/FastPitch_Adapter_Finetuning.ipynb:615]: I think for tutorial we don't need that large numbers? I have added a comment at the beginning `# Normally 200 epochs`

**Review by XuesongYang — APPROVED** [review]
Thanks for the fix!

**Pull request merged**